src/components/profile/view/ViewPersonalInfo.vue

```diff
@@ -50,21 +50,30 @@
         :officeLocation="staffInformation.officeLocation.value"
         :timezone="timezone.value"
       ></ProfileTeamLocation>
-      <MetaList v-if="this.staffInformation.staff.value">
+      <MetaList>
```

Comment:
Doesn't wrap on mobile (this was never good, but now gets worse). Maybe something like helps:
```css
display: grid;
grid-template-columns: repeat(auto-fill,minmax(10em, 1fr));
grid-gap: 1em;
```

Reply:
I pretty much went with this, just reduced the min a little, and used `auto-fit` so that the full width was always used.